### PR TITLE
Update rawInputPositions for Exchange operator

### PR DESCRIPTION
**Pull request by kewang1024**

### CODE REVIEW

**Comment by kewang1024** [comment]
Tested locally, originally
Java VS CPP
image
After fixing the rawInputPositions for exchange operator for CPP
image

File: velox/exec/Exchange.cpp [hunk 1]
```diff
@@ -117,12 +118,14 @@ RowVectorPtr Exchange::getOutput() {
       resultOffset = result_->size();
     }
   }
+  rawInputPositions += result_->size();
```
Comment (xiaoxmeng) [inline velox/exec/Exchange.cpp:121]:
```
const uint64_t rawInputPositions = result_->size();
```

**Review by xiaoxmeng — COMMENTED** [review]
@kewang1024 LGTM % nit.  Thanks!

**Comment by kewang1024** [comment]
Added in unit test to verifyExchangeStats.
Confirmed that without the change, the test would fail with: 
/Users/ke1024/presto/presto-native-execution/velox/velox/exec/tests/MultiFragmentTest.cpp:184: Failure
Expected: (exchangeOperatorStats.rawInputRows) > (0), actual: 0 vs 0

**Review by xiaoxmeng — APPROVED** [review]
@kewang1024 thanks for the update!

**Pull request closed without merging**